doc/telescope_changelog.txt

```diff
@@ -119,5 +119,16 @@ Guide to switch over to plenary.path
         before: require("telescope.path").read_file_async(filepath, callback)
         now:    require("plenary.path"):new(filepath):read(callback)
 
+                                                    *telescope.changelog-1406*
+
+Date: November 4, 2021
+PR: https://github.com/nvim-telescope/telescope.nvim/pull/1406
+
+Telescope requires Neovim release 0.5.1 or a recent nightly
+
+Due to making use of newly implemented extmark features, Telescope now
+requires users to be on Neovim 0.5.1 (the most recent stable version) or on a
+recent version of Neovim nightly (at most 1 week old).
```

Comment:
I'd like to keep this consistent with the README:

> Neovim (v0.5.1) or the latest neovim nighly commit is required for telescope.nvim to work.

which avoids the slippery slope that "at most 1 week old" may be.

Reply:
I see your point, but I also think that it is easy for a user to miss the word "latest", which then causes with confusion.
Being explicit about a timeframe helps to avoid that.

Maybe we could make the changelog say:

> ... or on the LATEST version of Neovim nightly.

and then leave the README as is?

Reply:
Sounds good :)